action_launcher.bash

```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-if [ -z "${INPUT_PULL_REQUEST_ID:-}" ]; then
```

Comment:
Not a bash expert but feels like the existing code is slightly "safer", i.e. no warning of "unbound variable" in this check if it hasn't been set.

What is the benefit of the introduced code aside of it being simpler?

Reply:
Hi @platisd

> Not a bash expert but feels like the existing code is slightly "safer", i.e. no warning of "unbound variable" in this check if it hasn't been set.

And why exactly is this good? Let's say we accidentally misspelled the name of this variable inside `action.yml` during the next edit. In the `master` branch we won't even notice it, while with this PR, such a typo will be detected immediately.

Reply:
Sounds reasonable